docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -12,7 +12,7 @@ run the commands described here. After the {agent} service is installed and runn
 make sure you run these commands without prepending them with `./` to avoid
 invoking the wrong binary.
 * Running {agent} commands using the Windows PowerShell ISE is not supported.
-* See also the <<elastic-agent-installation-minimum-requirements,minimum requirements>> described on this page. 
+* See also the <<elastic-agent-installation-minimum-requirements,minimum requirements>> described on this page.
```

Comment:
If we're changing from "minimum" to "resource", we should update the link target and its text too.

Reply:
The anchor was not changed, but you are right; it should be renamed, and to rename the reference.